src/Program.cs

```diff
@@ -161,6 +166,21 @@ private static IWebHost BuildHost()
                     }
                 });
 
+            string serviceName = "Ngsa.Application";
+            string serviceVersion = "1.1.0";
+            builder.ConfigureServices(services => services.AddOpenTelemetryTracing(b =>
```

Comment:
[Optional] Is it possible to lump ConfigureService call to previous statement in line 151?

```C#
WebHohst.CreateDefaultBuilder()
    .UseUrls()
    ...
    ...
    .ConfigureLogging()
    ...
    .ConfigureServices(service => services.AddOpenTelemetryTracing(...) ...)
```

This is just to be consistent with the rest of the code.